lib/rules/jsx-sort-props.js

```diff
@@ -44,6 +57,17 @@ module.exports = {
         // Whether alphabetical sorting should be enforced
         noSortAlphabetically: {
           type: 'boolean'
+        },
+        reservedFirst: {
```

Comment:
I believe that it (line 73) does do that. http://eslint.org/docs/developer-guide/working-with-rules